src/py42/sdk/__init__.py

```diff
@@ -192,13 +195,23 @@ def alerts(self):
 
     @property
     def auditlogs(self):
-        """A collections of methods for retrieving audit logs.
+        """A collection of methods for retrieving audit logs.
```

Comment:
observed a typo, not related to this feature, but creating a new PR for this wasn't worth it.